be/src/storage/update_manager.cpp

```diff
@@ -178,6 +182,24 @@ void UpdateManager::expire_cache() {
     }
 }
 
+void UpdateManager::evict_cache(int64_t memory_urgent_level, int64_t memory_high_level) {
+    int64_t capacity = _update_state_cache.capacity();
+    int64_t size = _update_state_cache.size();
+    int64_t memory_urgent = capacity * memory_urgent_level / 100;
+    int64_t memory_high = capacity * memory_high_level / 100;
+
+    if (size > memory_urgent) {
```

Comment:
It seems `memory_urgent` and `memory_high` have no diffenrence?

Reply:
> It seems `memory_urgent` and `memory_high` have no diffenrence?

memory_urgent is the memory_urgent_level but memory_high is memory_high_level, they are different.